lib/persistence/entities/talent.dart

```diff
@@ -65,3 +67,33 @@ class BaseTalent extends DTO {
     };
   }
 }
+
+class TalentTest extends DTO {
```

Comment:
kłóciłbym się że jest to trochę myląca nazwa (bo potem będziesz robił TalentTestTest), ale jest tak w oryginale więc niech będzie

Reply:
Nazwałbym to lepiej, ale tak to się "biznesowo" nazywa